server/sonar-server/src/test/java/org/sonar/server/platform/monitoring/EsMonitorTest.java

```diff
@@ -53,6 +54,7 @@ public void cluster_attributes() {
     assertThat(attributes.get("Number of Nodes")).isEqualTo(1);
   }
 
+  @Ignore
   @Test
   public void node_attributes() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Fix or remove this skipped unit test [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1607)